tox.ini

```diff
@@ -0,0 +1,15 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27
+
+[testenv]
+commands = nosetests {toxinidir}/tests []
+deps =
+    -rrequirements.txt
+    nose>=1.3
+    mock>=1.0.1
+    psutil==3.0.0
```

Comment:
FWIW, psutil >= 2.0.0 should be good enough here - the API is only incompatible for even older versions such as 1.3.1 as installed in my ubuntu system.